examples/algebraic_constraints.py

```diff
@@ -0,0 +1,44 @@
+import desbordante
+import pandas
+
+TABLE = 'datasets/cargo_march.csv'
+HEADER = 0
+SEPARATOR = ','
+P_FUZZ = 0.85
+FUZZINESS = 0.2
+BUMPS_LIMIT = 0
+WEIGHT = 0.1
+BIN_OPERATION = '-'
+AC_SEED = 11
+ITERATIONS_LIMIT = 4
+
+algo = desbordante.ACAlgorithm()
+
+df = pandas.read_csv(TABLE, sep=SEPARATOR, header=HEADER)
+df_without_id = df[['Delivery date', 'Dispatch date']]
+
+print("Algorithm needs the following data format:", end=' ')
+print(algo.get_needed_options())
+algo.load_data(df=df_without_id)
+
+print("Algorithm needs the following options:", end=' ')
+print(algo.get_needed_options())
+algo.execute(p_fuzz=P_FUZZ, fuzziness=FUZZINESS, bumps_limit=BUMPS_LIMIT, weight=WEIGHT,
+             bin_operation=BIN_OPERATION, ac_seed=AC_SEED, iterations_limit=ITERATIONS_LIMIT)
+
+ac_ranges = algo.get_ac_ranges()
+for ac_range in ac_ranges:
+    l_col = df.columns[ac_range.column_indices[0]]
+    r_col = df.columns[ac_range.column_indices[1]] 
+    print('Discovered ranges for', '(', l_col, BIN_OPERATION, r_col, ') are:')
+    print(ac_range.ranges)
+
+ac_exceptions = algo.get_ac_exceptions()
+for ac_exception in ac_exceptions:
+    print()
+    print('Row which is an exception, because it\'s difference is outside of discovered ranges:')
+    print(f'id: {df.loc[ac_exception.row_index][0]}')
+    print(f'Dispatch date: {df.loc[ac_exception.row_index][2]}')
+    print(f'Delivery date: {df.loc[ac_exception.row_index][1]}')
+    print(f'Difference: {df.loc[ac_exception.row_index][1] - df.loc[ac_exception.row_index][2]}')
```

Comment:
`df.iloc[ac_exception.row_index]`
It should be in a separate variable. Use `iloc` instead ([doc](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.loc.html)):
> Allowed inputs are:
> * A single label, e.g. 5 or 'a', (note that 5 is interpreted as a *label* of the index, and **never** as an integer position along the index).

Reply:
No, I meant `row = df.iloc[ac_exception.row_index]`

Reply:
Or even `id_, delivery_date, dispatch_date = df.iloc[ac_exception.row_index]`, if that's possible

Reply:
`id_, delivery_date, dispatch_date = df.iloc[ac_exception.row_index].tolist()` if the above doesn't work